test.sh

```diff
@@ -8,6 +8,7 @@
 # things if we do some refactoring.
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+chmod +x $DIR/seeds/*.*
```

Comment:
Rather than doing this, I've just been checking all the scripts into Git with the `x` bit set. Again, not a big deal.